### DISCUSSION TITLE
Support for RTL-SDR V4

### DISCUSSION THREAD

**Discussion by sadeepasepala** [jvde-github/AIS-catcher#156] [General]
Is AIS-Catcher currently support RTL-SDR V4 dongle?
When I used SDR# with updated driver I can see the AIS signal bursts at 161.975 and 162.025  I just tried with AIS catcher and got following but no decodes. 

C:\SDR>AIS-catcher.exe -N 8100 -o 4
AIS-catcher (build Sep  1 2023) v0.51-4-g47b6d64
(C) Copyright 2021-2023 jvde-github and other contributors
This is free software; see the source for copying conditions.There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Found Rafael Micro R828D tuner
TCP Server: start thread at port 8100
HTML Server running at port 8100
[R82XX] PLL not locked!
[R82XX] PLL not locked!

**Comment by OE1MWW** [comment]
In the RTL-SDR blog at https://www.rtl-sdr.com/ was a statement that the V4 will work 
only with a new driver - old driver is not compatible with V4. You will find a link for the 
'new' rtlsdr.dll on that page. 
RTL-SDR blog drivers are on GitHub:  https://github.com/rtlsdrblog/rtl-sdr-blog.
Download, unpack and copy the recent new rtlsdr.dll into your AISCatcher directory.
That should work with the V4. My V4 will arrive in two days, I can report, if you like.

**Reply by jvde-github** [reply]
A great, our messages crossed paths. Thank you! I just updated the RTLSDR that I ship for Windows with this new code. Best to use this one by using the Edge version (i.e. use to avoid compatibility issues - x64 vs x86 etc). Also, it resolves a few bugs in the standard RTL-SDR driver that manifests on Windows. 

Hopefully that fixes it but I do not have a V4 so cannot test.

**Reply by sadeepasepala** [reply]
Thanks OE1MWW, replacing the rtlsdr.dll worked too. now it doesen't give that PLL lock error. Also Jvde's new package worked out of the box too.. 
73

**Comment by jvde-github** [comment]
If you have one, did you receive AIS messages with a regular RTL-SDR dongle? At least we know it is a driver issue then.

The output you show looks as usual. However, I know that they updated the RTL-SDR driver to enable V4. This new code was not in the driver that I include in the AIS-catcher package but I have tried to merge it just now. Can you try again with the package from here:

https://github.com/jvde-github/AIS-catcher/releases/tag/Edge

J.

**Reply by sadeepasepala** [reply]
Hi Jvde,
Thank you very much for the quick response. I tried both .x64 and .x86 versions from above link with RTL-SDR V4 and both works like charm.  Thanks again. :)

**Reply by jvde-github** [reply]
Thanks for confirming! I have updated the Android app as well and will make a new release.

Thank you, Jasper
